MeshLib/MeshGenerators/VtkMeshConverter.cpp

```diff
@@ -455,6 +456,12 @@ void VtkMeshConverter::convertArray(vtkDataArray &array, MeshLib::Properties &pr
 		return;
 	}
 
+	if (vtkUnsignedLongArray::SafeDownCast(&array))
+	{
+		VtkMeshConverter::convertTypedArray<std::size_t>(array, properties, type);
```

Comment:
std::size_t -> unsigned long.
I'm not sure about the correctness of the assumption that std::size_t is always an unsigned long.